### DISCUSSION TITLE
Table Row Selection - Generating data

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#6390] [Help]
Hello,

I've been creating some forms using CRUD and I've come across one issue,
I created a CRUD form for a new table. That form is conected to a previous table (using data provider/table) where the products are stored, My intention is to select for example.. 5 products (by their product_id) and carry them over to the new table.
The problem is it won't allow me to create an entry in the new table because I'm storing multiple id's (i want multiple entries using the same date and category i selected in the form).

Also is there any automation I can create to copy data from table1 to table2, deleting it from table1 in the process using external data source like MySQL?

Very appreciated!

**Comment by melohagan** [comment]
Hi @insomniacj93 

I'm not quite grasping the issue. Would you be able to provide some screenshots please?
